lib/react-native/index.js

```diff
@@ -25,25 +25,35 @@ if (typeof Reflect !== "undefined" && Reflect.construct) {
   Reflect.construct.sham = 1;
 }
 
+const usingLegacyChromeDebugger = typeof DedicatedWorkerGlobalScope !== "undefined";
+
+if (usingLegacyChromeDebugger) {
+  throw new Error("This version of Realm JS doesn't support the legacy Chrome Debugger");
+}
+
 if (Platform.OS === "android") {
-  // Getting the native module on Android will inject the Realm global (on Android)
+  // Getting the native module on Android will inject the Realm global
+  // eslint-disable-next-line no-unused-vars
   const RealmNativeModule = NativeModules.Realm;
-  console.log({ RealmNativeModule, Realm: global.Realm });
 }
 
+// TODO: Remove the need to store Realm as a global
+// @see https://github.com/realm/realm-js/issues/2126
+const Realm = globalThis.Realm;
+
 // Otherwise, we must be in a "normal" react native situation.
 // In that case, the Realm type should have been injected by the native code.
 // If it hasn't, the user likely forgot to install the RealmJS CocoaPod
-if (typeof global.Realm === "undefined") {
+if (typeof Realm === "undefined") {
   throw new Error(
     'Missing Realm constructor. Did you run "pod install"? Please see https://realm.io/docs/react-native/latest/#missing-realm-constructor for troubleshooting',
```

Comment:
This URL ends up in a 403.
This error message isn't mentioned in our current documentation, but can be found in the [legacy documentation](https://docs.mongodb.com/realm-legacy/docs/javascript/3.6.0/index.html) and in [this blog post](https://www.mongodb.com/developer/how-to/build-offline-first-react-native-mobile-app-with-expo-and-realm/).

Reply:
I agree, it does make sense to update the URL in this PR too.

Reply:
I think we should remove the link altogether.  The docs site is no longer something we control, so the URL may change at any time..